cdci_data_analysis/analysis/parameters.py

```diff
@@ -761,10 +739,8 @@ def __init__(self,value=None,units='sigma',name=None):
                                    name=name,
                                    allowed_units=_allowed_units)
                                    #wtform_dict=wtform_dict)
-
-
-
-
+    # TODO this method, behaves exaclty the same as the one for the Energy and SpectralBoundary classes
```

Comment:
this should be almost the same. It behaves like `Float`. The only difference is units (`sigma` in this case).